assets/scss/_grpc.scss

```diff
@@ -229,10 +228,9 @@ c - Component (Aware of its content/context...)
 // Object
 .o-banner {
   width: 100%;
-  position: fixed;
-  margin-left: -15px;
+  position: absolute;
   padding-top: 15px;
-  z-index: 32;
+  z-index: $z-index-navbar - 1;
```

Comment:
nit: Please add a single line comment just above this line explaining why we need to ensure the z-index for the banner is less than the navbar.